pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
   flutter: "3.28.0-0.1.pre"
 
 dependencies:
+  app_links: ^6.3.3
```

Comment:
Is an external plugin required to support deep links? I think not, according to flutter documentation. It would be better to do without imo.

Reply:
Hmm I'm not sure, the flutter docs recommend this package as an alternative to using go_router (see note here https://docs.flutter.dev/cookbook/navigation/set-up-app-links#2-modify-androidmanifest-xml)

I'll check if there's a way to get the URI Stream without a plugin

Reply:
Yeah I thought app_links is needed to actually get access to the link URL - haven't checked yet if there is a native way do get it, there probably is, I'll have a look

Reply:
Right, found the way to do it without dependencies.

I only added the implementation for Android (as I'm neither familiar with Swift/iOS, nor do I have the hardware to actually test this)